src/scse/controller/miniscot.py

```diff
@@ -164,12 +163,10 @@ def reset_agents(self, context, state):
                 )+" reset"] = module_end_time - module_start_time
                 self._miniscot_time_profile[module.get_name(
                 )+" compute_actions"] = 0
-        self._metrics.reset(context, state, self._battery_penalty)
+        self._metrics.reset(context, state)
         logger.debug("trying to reset signed-in services")
         registry.reset_signed_in_services(context)
 
-        self.episode_reward += self._metrics._upfront_battery_cost
```

Comment:
@collinskatie We add an upfront battery cost for each batter in `national_grid_cash_accounting.py` - I think we were accidentally adding the cost of an additional battery here?